training/tensorflow/custom_containers/hypertune/resnet/hypertune_hook.py

```diff
@@ -0,0 +1,32 @@
+"""Communicate evaluation results to hyperparmeter tuning servicer."""
```

Comment:
nit: Need Copyright header

Reply:
@gogasca , thank you for the review. 
I did amend the commit and pushed. 
Can you see the changes? 
do I need to create another pull request with it?

Reply:
No need to create a new PR. Thanks for this contribution!